packages/Webkul/Shop/src/Http/Controllers/Customer/AddressController.php

```diff
@@ -89,18 +82,9 @@ public function store(AddressRequest $request)
      */
     public function edit($id)
     {
-        $customer = auth()->guard('customer')->user();
-
-        $address = $this->customerAddressRepository->findOneWhere([
-            'id'          => $id,
-            'customer_id' => $customer->id,
-        ]);
+        $address = $this->customerAddressRepository->find(auth()->guard('customer')->id());
 
-        if (! $address) {
-            abort(404);
-        }
-
-        return view($this->_config['view'], array_merge(compact('address'), [
+        return view('shop::customers.account.addresses.edit', array_merge(compact('address'), [
```

Comment:
Remove compact and array_merge function and pass simple array with key-value pair.

Reply:
Fixed